plan/plan.go

```diff
@@ -263,7 +263,7 @@ type PhysicalPlan interface {
 	ToPB(ctx context.Context) (*tipb.Executor, error)
 
 	// getPushedProp tries to push the required property to its child and get the result property.
```

Comment:
child -> children?
Update this comment.
And this function name may also need to update.